### ISSUE TITLE
Can we support a list of bulk IP addresses?

### ISSUE THREAD

**Issue by kinz-dev** [rryuliu/WizLightControl#2]
Enhancement requests:  can we support a list of IP addresses to control the bulk at the same time?

**Comment by rryuliu** [comment]
> Enhancement requests:  can we support a list of IP addresses to control the bulk at the same time?

Im sorry, this app is just a demo. I'm still learning Swift, so I don't have any plans to support more light bulbs at the moment...:(

**Comment by Dheeraj-FP** [comment]
Hey Man this is a great application. The light brightness controller and the change from warm to white controller is not working. But everything else is great. 

Like Kinz suggested, it will be great if you can provide an option to control multiple lights.

**Comment by rryuliu** [comment]
> Hey Man this is a great application. The light brightness controller and the change from warm to white controller is not working. But everything else is great.
> 
> Like Kinz suggested, it will be great if you can provide an option to control multiple lights.

Hi Dheeraj, I tested it, and currently, everything is working fine. I couldn't reproduce the bug you encountered.
Perhaps the issue is specific to the model of your lamp, which might be very new (or very old).
Unfortunately, I only have one model of the lamp available for testing, so I'm sorry that I may not be able to resolve this problem for the time being.

**Comment by Dheeraj-FP** [comment]
alright no worries man. Just a quick question. i did take a duplicate of the application and had 2 instances run simultaneously. Idea behind it was to assign separate ips for each instance for controlling the bulbs separately. But what happened was if i change it in one instance, it gets applied to the other one also. Is there a way where i can separate the app and two instances separately?

**Comment by rryuliu** [comment]
> alright no worries man. Just a quick question. i did take a duplicate of the application and had 2 instances run simultaneously. Idea behind it was to assign separate ips for each instance for controlling the bulbs separately. But what happened was if i change it in one instance, it gets applied to the other one also. Is there a way where i can separate the app and two instances separately?

Try changing the "lightIP" in line 12 of "MainView.swift" to another name in one of the instances and see if it works.
 
like this:
`@AppStorage("lightIp2") var lightIp: String = ""`

**Comment by Dheeraj-FP** [comment]
Thanks for the swift response. i used the lightcontrol release package that was available here. I couldn't find this file in it. But i was able to find it on the source code zip. Should i change this code and take another build?

**Comment by rryuliu** [comment]
> Thanks for the swift response. i used the lightcontrol release package that was available here. I couldn't find this file in it. But i was able to find it on the source code zip. Should i change this code and take another build?

Yes, if you want to configure different light bulbs for two separate instances, 
you need to download the project files and build a version using Xcode to meet your requirements.

**Comment by Dheeraj-FP** [comment]
Thanks, man. I will try it out and let you know.

**Comment by davidraaamos** [comment]
> Thanks, man. I will try it out and let you know.

Let us know if it worked for you and how you did it. I have 0 knowledge and would love to use it.
Thanks :)
